### ISSUE TITLE
No Tool Settings tab on Linux

### ISSUE THREAD

**Issue by claytron5000** [flameshot-org/flameshot#2121]
### Flameshot Version

Flameshot 0.5.1-1(Debian)

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Ubuntu

### Description

When taking a screenshot there is no Tool Settings tab. In fact, I didn't know there was a Tool Settings tab until I used Flameshot on MacOS. I've also seen this behavior on PopOS (ubuntu based distro)

### Steps to reproduce

1. On Linux
2. Install Flameshot
3. Click flameshot tray icons
4. Expected: Screen is darkened, prompt show ups, Tool Settings are available on the site
5. Actual: Screen is darkened, prompt show ups, Tool Settings are not visible (NOTE, they don't appear after defining a capture area either.)

### System Information

Linux t420 5.4.0-81-generic #91~18.04.1-Ubuntu SMP Fri Jul 23 13:36:29 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

KDE (also observed in Gnome)

x11

**Comment by borgmanJeremy** [comment]
Please upgrade to at least version 0.10.1, we no longer maintain 0.5

**Comment by claytron5000** [comment]
I'll update someway, but it looks like this is the latest available to the ubuntu source.
I'm not sure how that gets updated.
```
Package: flameshot
Version: 0.5.1-2
Priority: optional
Section: universe/graphics
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: Boyuan Yang <073plan@gmail.com>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 823 kB
Depends: libc6 (>= 2.14), libgcc1 (>= 1:3.0), libqt5core5a (>= 5.9.0~beta), libqt5dbus5 (>= 5.0.2), libqt5gui5 (>= 5.7.0), libqt5network5 (>= 5.0.2), libqt5widgets5 (>= 5.7.0), libstdc++6 (>= 5.2)
Suggests: ca-certificates, openssl
Homepage: https://github.com/lupoDharkael/flameshot
Download-Size: 217 kB
APT-Manual-Installed: yes
APT-Sources: http://us.archive.ubuntu.com/ubuntu bionic/universe amd64 Packages
Description: Powerful yet simple-to-use screenshot software
 Flameshot is a powerful yet simple-to-use screenshot software.
 Notable features include customizable appearance, in-app screenshot editing,
 D-Bus interface, experimental GNOME/KDE Wayland support, integration with
 Imgur and support for both GUI and CLI interface.
```

**Comment by claytron5000** [comment]
Same version when installed through the GUI software installer.

**Comment by borgmanJeremy** [comment]
Yes it's because your version of Ubuntu is old. You can manually download the package from our release page or use flatpak or snap.

**Comment by mmahmoudian** [comment]
@claytron5000 please follow one if the options presented in this guide:

https://flameshot.org/guide/installation/installation-linux/

**Comment by claytron5000** [comment]
Not to be a stickler, but Ubuntu 18 is still in standard support until April 2023. https://ubuntu.com/about/release-cycle

I have installed the application from source on another computer and I’ll do it again if I have to (I really like flameshot). 

I don’t know what kind of effort is involved in maintaining apt repositories, but I’d respectfully ask that maintained distributions get application updates. 

I am updating to Ubuntu 20, but I’d expect there are users out there who aren’t, and might like to use flameshot.

Thank you for your time.

**Comment by borgmanJeremy** [comment]
This is a common confusion, but we as the application developers have no input on the software in a given distro. So there is literally nothing we can do for you besides provide something like flatpak where we control the release not the distro maintainer's.

Furthermore you are using a static or "stable" distro. By design the Ubuntu devs are going to keep the same version that was released at the launch of 18.04 unless the back port a security fix. This is feature of an LTS.

**Comment by borgmanJeremy** [comment]
Also we actually go out of our way to make sure we build against the latest to LTSs. So when 22.04 comes out we will drop 18.04 , but not before then. 

The code has specific build flags to keep compatibility.

**Comment by claytron5000** [comment]
Thanks for the explanation. I upgraded to 20.04. The available package for that release is 0.6, which has the same issue (no tool setting tab.) I installed the 0.10 release, which does work. I wonder if it would make sense to update the documentation. As a user I initially thought the application was not usable.

**Comment by mmahmoudian** [comment]
@claytron5000 what distro maintainers do and pick for their distros are mot influenced by us. If you want a more updated version from the official repo of your linux distribution, you should contact them. But if you want to use the latest Flameshot version and you are okay with using Flatpak, Snap, or even AppImage, you can install Flameshot from those.

Also about documentation, soon we will roll out our new website which will have some information
